### PR TITLE
Create all link view

### PR DESCRIPTION
**Pull request by HansolLee1993**
Created sidebar link views.
Removed dashboard and replaced with equivalent add expense page.
Fixed controller routes.

### CODE REVIEW

File: resources/views/statistics.blade.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@extends('layouts.app')
+@section('content')
+    <div class="main-panel">
+        <div class="container-fluid">
+            <h4>{{ Auth::user()->firstname}}'s expenses</h4>
```
Comment (brynbeaudry) [inline resources/views/statistics.blade.php]: Was this intended?
Maybe user's Statistics. Wdyt?
http://picpaste.com/pics/stats-VgwBBfIT.1490755551.PNG

File: resources/views/tableList.blade.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@extends('layouts.app')
+@section('content')
+    <div class="main-panel">
+        <div class="container-fluid">
+            <h4>{{ Auth::user()->firstname}}'s expenses</h4>
```
Comment (brynbeaudry) [inline resources/views/tableList.blade.php]: Same here, wdyt?

File: routes/web.php [hunk 3]
```diff
@@ -13,24 +13,46 @@
...
 
-  Route::get('/friends/search', 'FriendsController@search_view');
+
+
+  /* FriendController */
+  //Route::get('/friends/search', 'FriendsController@search_view');
 
   //TODO please implement the controller for this BE. Feel free to change the URIs if needed.
 
+  /* statisticsController */
```
Comment (brynbeaudry) [inline routes/web.php]: 👍

**Comment by brynbeaudry** [comment]
dashboard2
@danielcapacio @ConnorJang @HansolLee1993 

@itsmattli 

> what do you think about leaving the add expenses as a side bar button that can be clicked at any time to open the modal.

I think it should be in the nav because the user will be using it so much

>  I think that having the add expenses on the side linking to a page that shows your expenses and the add maybe we can change the two.
Yes, as in the image I think that there should be a page for "My Expenses" and a seperate one for Dashboard, which is your summary

> TL;DR: add expense to nav bar, can be clicked at any time to add an expense. Current add expense button converted to my expenses, just showed everything under the current add expense button on add expense view

I totally agree with that first sentence. Totally agree with second sentence.

Does anyone on the front end agree with that? @HansolLee1993 will you be able to make the name changes to the sidebar and make it so that the green styling goes on the selected tab? Then I'll push this one to staging

**Comment by danielcapacio** [comment]
Sweet, I like the idea of moving the add expense modal function up to the nav bar; it'll be easily accessible and non-tedious for the user. The green-highlighting part seems pretty doable as well, so I'm on board with that. Also, the mockup is great - illustrates the approach perfectly.

Great input Bryn and Matt!

**Comment by nikoootine** [comment]
@SplitSet3R/front-end  can we get this merge soon

File: app/Http/Controllers/ProfileController.php [hunk 4]
```diff
@@ -114,11 +114,19 @@ public function index($profile_name) {
...
             return view('debug', compact('ststus'));
             //TODO: this should return to a view where a limited profile is shown displaying either pending or not friends
         }
+
+
+    }
+
+    public function userProfile(){
+        return view('userProfile');
+
```
Comment (nikoootine) [inline app/Http/Controllers/ProfileController.php]: Do we need this method?

**Comment by nikoootine** [comment]
Closing pull request as the changes requested are in another branch
